docs/backend-system/architecture/01-index.md

```diff
@@ -57,7 +57,7 @@ Just like plugins, modules also have access to services and can depend on their
 
 A detailed explanation of the package architecture can be found in the
 [Backstage Architecture
-Overview](../../overview/architecture-overview.md#package-architecture). The
+Overview](../../overview/architecture-overview/#package-architecture). The
```

Comment:
The link is correct, it should be the `.md` file but it's the line break above that is causing this issue. Not sure why but Docusaurus does not like line breaks in links :(

Reply:
Thanks @awanlin, you are completely right. I assumed the ".md" was the issue without even test it, sorry. I just tested locally and it is the new line the issue, as you said. It is now fixed.

Reply:
No worries, it's an odd issue so nothing to stress about. 👍